src/main/java/com/sourcegraph/cody/agent/CodyAgentClient.java

```diff
@@ -63,8 +65,19 @@ public void webviewPostMessage(WebviewPostMessageParams params) {
         && extensionMessage.getType().equals(ExtensionMessage.Type.TRANSCRIPT)) {
       ApplicationManager.getApplication().invokeLater(() -> onNewMessage.accept(params));
     } else {
-      logger.debug("onNewMessage is null or message type is not transcript");
-      logger.debug(String.format("webview/postMessage %s: %s", params.getId(), extensionMessage));
+      var listener = this.webviewMessageListeners.get(params.getId());
+      if (listener != null) {
+        listener.accept(params.getMessage());
+      } else {
+        logger.warn(
```

Comment:
Do you think it makes sense to change it back to warn?
I recently switched if from warn to debug because I think this is not a real warning (we consciously do not handle other messages). I think we should setup it in the way that we on dev machines [always see all debug logs](https://plugins.jetbrains.com/docs/intellij/ide-infrastructure.html#logging) but it does not leak to the users.